en/Plugins/Getting started/Use Svelte in your plugin.md

```diff
@@ -110,7 +110,7 @@ In the root directory of the plugin, create a new file called `Counter.svelte`:
 
 ## Mount the Svelte component
 
-To use the Svelte component, it needs to be mounted on an existing [[HTML elements|HTML element]]. For example, if you are mounting on a custom [[ItemView|ItemView]] in Obsidian:
+To use the Svelte component, it needs to be mounted on an existing [[HTML elements|HTML element]]. For example, if you are mounting on a custom [[ItemView|ItemView]] in Obsidian, create a file called `example_view.ts`
```

Comment:
Other places in the docs use Pascal case, so `ExampleView.ts`, as the file naming convention.